src/applications/yellow-ribbon/containers/YellowRibbonApp/index.jsx

```diff
@@ -103,21 +103,21 @@ export const YellowRibbonApp = ({ hasFetchedOnce }) => (
 
           {hasFetchedOnce && (
             <>
-              <h3>Search criteria</h3>
-              <p>
+              <h3 className="vads-u-margin-top--2">Helpful links</h3>
```

Comment:
Do we need a test of some sort to confirm that these links are indeed "helpful?"

Reply:
Hmm... so not a unit test... not an e2e test... but a helpfulness test... so a whole new kind of test that sends out an email to all users each time it's run asking if they think this feature is indeed helpful 🤔hahaha